documentation/library_guide/cmake_support.rst

```diff
@@ -0,0 +1,77 @@
+CMake Support
+#############
+
+General Usage
+=============
+`CMake <https://cmake.org/cmake/help/latest/index.html>`_ is a cross-platform build system generator.  It can be used to generate build scripts which can then be used to build and link a users application.
+
+``oneDPLConfig.cmake`` and ``oneDPLConfigVersion.cmake`` are distributed with |onedpl_short|.  These files allow to integratration of |onedpl_short| into user projects with the `find_package <https://cmake.org/cmake/help/latest/command/find_package.html>`_ command. Successful invocation of ``find_package(oneDPL <options>)`` creates imported target `oneDPL` that can be passed to the `target_link_libraries <https://cmake.org/cmake/help/latest/command/target_link_libraries.html>`_ command.
+
+Some useful CMake variables (`here <https://cmake.org/cmake/help/latest/manual/cmake-variables.7.html>`_ you can find a full list of CMake variables for the latest version):
+
+- `CMAKE_CXX_COMPILER <https://cmake.org/cmake/help/latest/variable/CMAKE_LANG_COMPILER.html>`_ - C++ compiler used for build, e.g. ``CMAKE_CXX_COMPILER=dpcpp``.
+- `CMAKE_BUILD_TYPE <https://cmake.org/cmake/help/latest/variable/CMAKE_BUILD_TYPE.html>`_ - build type that affects optimization level and debug options, values: ``RelWithDebInfo``, ``Debug``, ``Release``, ...; e.g. ``CMAKE_BUILD_TYPE=RelWithDebInfo``.
+- `CMAKE_CXX_STANDARD <https://cmake.org/cmake/help/latest/variable/CMAKE_CXX_STANDARD.html>`_ - C++ standard, e.g. ``CMAKE_CXX_STANDARD=17``.
+
+|onedpl_short| Backend Selection
+==============
+
+The |onedpl_short| parallel backend is selected based on compiler and environment availability, in combination with the user defined ``ONEDPL_PAR_BACKEND`` option. ``ONEDPL_PAR_BACKEND`` is optional, and has three possible valid options: ``tbb`` (oneTBB), ``openmp`` (OpenMP), and ``serial``.  If SYCL is supported by the compiler, ``dpcpp`` backend is always selected.  If this ``ONEDPL_PAR_BACKEND`` is not set then the first suitable backend is chosen among oneTBB, OpenMP and serial, in that order.  |onedpl_short| is considered as not found (``oneDPL_FOUND=FALSE``) if ``ONEDPL_PAR_BACKEND`` is specified, but not found or not supported.
```

Comment:
This is optional. You can consider adding two tables.

**Backend for Parallel Execution Policies (par and par_unseq)**

ONEDPL_PAR_BACKEND | TBB | OpenMP | Serial
-- | -- | -- | --
[not set] | oneDPL heuristics |  oneDPL heuristics | oneDPL heuristics
tbb | + |   |  
openmp |   | + |  
serial |   |   | +

The heuristics is the following: the first suitable backend is chosen among oneTBB, OpenMP and serial, in that order.

It would be better merge cells with "oneDPL heuristics" but it seems imposible with github markdown. 

**Backend for Device Execution Policies**

DPCPP |
-- |
oneDPL heuristics

The heuristics is the following: DPCPP backend is enabled if the compiler supports -fsycl option and SYCL headers are available.


We are considering adding a switch for dpcpp (device) backend selection. The second table may evolve to that in the future:
**Backend for Device Execution Policies**

ONEDPL_DEVICE_BACKEND | DPCPP
-- | --
[not set] | oneDPL heuristics
dpcpp | +
off |  